galaxy/templates/priorityclass-job.yaml

```diff
@@ -5,8 +5,12 @@ metadata:
   name: {{ include "galaxy.pod-priority-class" . }}
   labels:
     {{- include "galaxy.labels" . | nindent 4 }}
-value: -1000
+value: {{ if .Values.jobs.priorityClass.value -}} 
+       {{ .Values.jobs.priorityClass.value}} 
+       {{- else -}} 
+       -1000 
+       {{- end}}
 preemptionPolicy: Never
 globalDefault: false
-description: "Galaxy jobs will run when cluster resources are available and will not preempt existing workloads."
+description: "By default, Galaxy jobs will not preempt existing workloads but will run when cluster resources are available. Depending on the cluster setup, the preemption value can be modified to prompt scaling of the cluster for the computation."
```

Comment:
I think some guidance on what values would elicit the desired behaviour in the autoscaler case would also be helpful.

Reply:
Thank you, sounds great!